src/main/scala/org/allenai/deep_qa/experiments/SciQDatasetExperiments.scala

```diff
@@ -0,0 +1,302 @@
+      ("data files" -> Seq(ScienceDatasets.readingComprehensionOmnibusQaGradeFourTestQuestionsWithBuscBackground))
+
+  /*
+   * Create combined datasets with Omnibus4Train + SciQ Train and
+   * Omnibus8Train + SciQ Train.
+   */
+
+  val combinedSciQTrainOmnibusEightTrainDataset: JValue =
+    ("dataset type" -> "combined") ~
+      ("datasets" -> Seq(sciQTrainDataset, omnibusEightTrainReadingComprehensionDataset))~
+      ("output directory" -> s"/efs/data/dlfa/processed/omnibus_8_train_and_sciq_train_combined/")
+
+  val combinedSciQTrainOmnibusFourTrainDataset: JValue =
+    ("dataset type" -> "combined") ~
+      ("datasets" -> Seq(sciQTrainDataset, omnibusFourTrainReadingComprehensionDataset))~
+      ("output directory" -> s"/efs/data/dlfa/processed/omnibus_4_train_and_sciq_train_combined/")
+
+  def omnibusGradeFourExperiment(
+    name: String,
+    modelParams: JValue,
+    trainingDataset: JValue=omnibusFourTrainReadingComprehensionDataset
+  ): JValue = {
+    experiment(
+      name,
+      modelParams,
+      trainingDataset,
+      omnibusFourDevReadingComprehensionDataset
+    )
+  }
+
+  def omnibusGradeEightExperiment(
+    name: String,
+    modelParams: JValue,
+    trainingDataset: JValue=omnibusEightTrainReadingComprehensionDataset
+  ): JValue = {
+    experiment(
+      name,
+      modelParams,
+      trainingDataset,
+      omnibusEightDevReadingComprehensionDataset
+    )
+  }
+
+  def combinedSciQOmnibusGradeFourExperiment(
+    name: String,
+    modelParams: JValue,
+    trainingDataset: JValue=combinedSciQTrainOmnibusFourTrainDataset
+  ): JValue = {
+    experiment(
+      name,
+      modelParams,
+      trainingDataset,
+      omnibusFourDevReadingComprehensionDataset
+    )
+  }
+
+  def combinedSciQOmnibusGradeEightExperiment(
+    name: String,
+    modelParams: JValue,
+    trainingDataset: JValue=combinedSciQTrainOmnibusEightTrainDataset
+  ): JValue = {
+    experiment(
+      name,
+      modelParams,
+      trainingDataset,
+      omnibusEightDevReadingComprehensionDataset
+    )
+  }
+
+  val attentionSumReader: JValue = {
+    Models.attentionSumReader merge
+    (("patience" -> 1) ~
+      ("preferred_backend" -> "theano") ~
+      ("encoder" ->
+        ("default" ->
+          ("type" -> "bi_gru")~
+          ("output_dim" -> 384)
+        )
+      ) ~
+      ("seq2seq_encoder" ->
+        ("default" ->
+          ("type" -> "bi_gru") ~
+          ("encoder_params" ->
+            ("output_dim" -> 384)
+          ) ~
+          ("wrapper_params" -> JObject())
+        )
+      ) ~
+      ("optimizer" ->
+        ("type" -> "adam") ~
+        ("clipnorm" -> 10.0) ~
+        ("lr" -> 0.0005)
+      ) ~
+      ("embedding_dropout" -> 0.0) ~
+      ("patience" -> 0) ~
+      ("embedding_size" -> 256) ~
+      ("num_epochs" -> 5)
+    )
+  }
+
+  val gatedAttentionReader: JValue = {
+    Models.gatedAttentionReader merge
+    (("embedding_size" -> 100) ~
```

Comment:
I would put all of these default settings into `Models.gatedAttentionReader`, or something similar inside `Models`.